invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/index.js

```diff
@@ -406,14 +405,15 @@ class CommunityProfileForm extends Component {
         };
       }
     );
-
     // Serialize each funding record, award being optional.
-    const funding = submittedCommunity.metadata?.funding.map((fund) => {
-      return {
-        ...(fund.award && { award: serializeFunding(fund.award) }),
-        funder: serializeFunding(fund.funder),
-      };
-    });
+    const funding =
+      submittedCommunity.metadata?.funding &&
```

Comment:
this is redundant as we use the `?.`...is there an issue?

Reply:
thanks!